coldfront/plugins/qumulo/utils/qumulo_api.py

```diff
@@ -210,10 +210,11 @@ def update_allocation(
         name: str = None,
         limit_in_bytes: int = 0,
     ):
-        if not protocols:
-            raise ValueError("protocols should be defined.")
 
-        self.validate_protocols(protocols=protocols)
+        if protocols:
+            self.validate_protocols(protocols=protocols)
```

Comment:
@peaches7083 
The `validate_protocols` always handled an empty list of protocols, not raising an exception.